<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>875</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>877</width>
    <height>553</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>–ü–æ–¥–∫–ª—é—á–∏—Ç—å—Å—è</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portlineEdit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>420</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="timelineEdit">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>390</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>470</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>–°—Ç–∞—Ä—Ç</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>–û—Å—å</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="xAxisButton">
       <property name="text">
        <string>X</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+Z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="yAxisButton">
       <property name="text">
        <string>Y</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+X</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="xyAxisButton">
       <property name="text">
        <string>XY</string>
       </property>
       <property name="shortcut">
        <string>Ctrl+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>–ü–æ—Ä—Ç</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="portComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshportButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>25</horstretch>
         <verstretch>25</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>25</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string>üîÉ</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>231</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>–í—ã–±–æ—Ä –ö–∞–Ω–∞–ª–∞</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="chanelcomboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>611</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="graph_layout"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
